fuse/plugins/detector_physics/electron_drift.py

```diff
@@ -168,6 +168,12 @@ class ElectronDrift(FuseBasePlugin):
         help="Field distortion map used in fuse (Check if we can remove _fuse from the name)",
     )
 
+    liquid_level = straxen.URLConfig(
+        default="take://resource://" "SIMULATION_CONFIG_FILE.json?&fmt=json" "&take=liquid_level",
```

Comment:
I guess the formatting was done by pre-commit but I would propose to remove the unnecessary `" "` in the string for the default value.

Reply:
Done